README.md

```diff
@@ -1 +1,51 @@
 # project-liberation
+
+## Development setup
```

Comment:
Also please before `Development setup` please add some information about Python, Django and npm versions.